ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -239,16 +240,17 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "batched_uploader_functio
   resource_group_name = var.rg_name
   severity            = var.severity
   frequency           = 5
-  time_window         = 7
+  time_window         = 15
   enabled             = contains(var.disabled_alerts, "batched_uploader_function_not_triggering") ? false : true
 
   data_source_id = var.app_insights_id
 
   query = <<-QUERY
 requests
-${local.skip_on_weekends}
-| where timestamp >= ago(7m) 
+| where timestamp >= ago(15m) 
     and operation_Name =~ 'QueueBatchedReportStreamUploader'
+    and dayofweek(now()) between (time(1) .. time(5))
+    and hourofday(now()) between (time(14) .. time(22))
```

Comment:
whats the time-zone for this? also is the time window 2pm to 10pm?

Reply:
As mentioned in the Additional Information section, the time zone for this is UTC, with the window for business hours centered on the east coast. The time window would be from 1400-2200 UTC, which would be from 1000-1800 EDT or 0900-1700 EST. That should cover us for DST.